packages/bun-landing/page.tsx

```diff
@@ -715,10 +715,6 @@ export default {
           </p>
         </div>
       </section>
-
-      <section id="explain-section">
-        <div id="explain"></div>
-      </section>
```

Comment:
Can you add a screenshot of before removing and after removing this section, want to make sure it doesn't change any styling